### ISSUE TITLE
Visual bug: textures missing on Fire Plume Ridge Hot Spot for quest Finding the Source

### ISSUE THREAD

**Issue by akhial** [dalaranwow/dalaran-wow#5215]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

The Fire Plume Ridge Hot Spot is shown as a dark gray 3d model with no textures instead of showing lava.

**Expected behaviour**: Tell us what should happen instead

Should show correct textures.

**Steps to reproduce the problem**:

1. Go to Un'Goro and stand next to a Fire Plume Ridge Hot Spot.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

http://www.wowhead.com/quest=974/finding-the-source

**Include Screenshots from the issue if necessary**

![bug](https://user-images.githubusercontent.com/20596622/34456125-4c99d9b6-ed8f-11e7-9c7d-8ee58055dd46.jpg)

**Comment by akhial** [comment]
Here is how the hot spot looked in retail

![real](https://user-images.githubusercontent.com/20596622/34456296-41f53952-ed93-11e7-9102-cda9d93c959d.jpg)

**Comment by TeridanDW** [comment]
Hello and thank you for the report @akhial.
I don't think this report is valid because the model of the object was updated after wrath. Here you can see a video from retail wrath and the visuals are the same.
https://www.youtube.com/watch?v=a4owkL07cqs

> This video was created before Cataclysm was released.

**Comment by akhial** [comment]
Oh ok nice! Sorry for the mistake XD
